### ISSUE TITLE
Step 27 of this document is incorrect

### ISSUE THREAD

**Issue by ericd-netapp** [NetAppDocs/ontap-systems-upgrade#68]
PS engineer was following this procedure and we got into a boot loop because he set the following value as per the documentation

setenv bootarg.storageencryption.support true

On step27 it states, "If NetApp Storage Encryption (NSE) is in use on this configuration, the setenv bootarg.storageencryption.support command must be set to true"

Actually, that is not completely correct.  Yes, his system was using Encryption, but not NSE or SED disks.

As per this KB on Step 7:  https://kb.netapp.com/Advice_and_Troubleshooting/Data_Storage_Software/ONTAP_OS/How_to_repurpose_an_HA_Pair_with_ONTAP_9

The question is better stated as this:

"Is the HA pair using Self-encrypting FIPS-certified SAS or NVMe drives (FIPS drives)?"

Long story short, when you set this value wrong on a system without these drives it will cause it to boot loop.

I'm 99.9999 percent sure I'm correct here. 

We should also consider having them run a "set-defaults" on the new nodes too.  (to match the KB)

Page: [Install and boot node3](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade-arl-auto-app/install_boot_node3.html)

**Comment by netapp-pcarriga** [comment]
@ericd-netapp Hi Eric, the updates are now published which include information on how to determine if your system uses self-encrypting drives and on the types self-encrypting of drives support by ONTAP  The instructions for setting the "bootarg.storageencryption.support" and "kmip.init.maxwait" variables in Step 27 have also been to for the types of self-encrypting that are in use. You can view these updates using the following links:

[Prepare the nodes for upgrade](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade-arl-auto-app/prepare_nodes_for_upgrade.html#correct-aggregate-ownership-if-an-arl-precheck-failsl) > Step5
[Install and boot node3](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade-arl-auto-app/install_boot_node3.html) > Step 27
[Install and boot node4](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade-arl-auto-app/install_boot_node4.html) > Step 27
Thanks again for providing feedback and helping to improve the documentation,
